exercises/leap/canonical-data.json

```diff
@@ -1,37 +1,22 @@
 {
    "cases": [
       {
-         "description": "leap year in twentieth century",
-         "input": 1996,
-         "expected": true
-      },
-      {
-         "description": "odd standard year in twentieth century",
-         "input": 1997,
-         "expected": false
-      },
-      {
-         "description": "even standard year in twentieth century",
-         "input": 1998,
+         "description": "year not divisible by 4: not a leap year",
```

Comment:
Agreeing with Wikipedia: "A year that is not a leap year is called a common year." is a good idea.
"not a leap year" is a little too self referential.